packages/workbox-build/src/inject-manifest.ts

```diff
@@ -135,27 +135,27 @@ export async function injectManifest(config: unknown): Promise<BuildResult> {
   try {
     swFileContents = await fse.readFile(options.swSrc, 'utf8');
   } catch (error) {
-    throw new Error(`${errors['invalid-sw-src']} ${error.message}`);
+    throw new Error(`${errors['invalid-sw-src']} ${error instanceof Error && error.message ? error.message : ''}`);
   }
 
   const injectionResults = swFileContents.match(globalRegexp);
   if (!injectionResults) {
     // See https://github.com/GoogleChrome/workbox/issues/2230
+    const injectionPoint = options.injectionPoint ? options.injectionPoint : '';
     if (upath.resolve(options.swSrc) === upath.resolve(options.swDest)) {
-      throw new Error(errors['same-src-and-dest'] + ' ' +
-        options.injectionPoint);
+      throw new Error(`${errors['same-src-and-dest']} ${injectionPoint}`);
     }
-    throw new Error(errors['injection-point-not-found'] + ' ' +
-      options.injectionPoint);
+    throw new Error(`${errors['injection-point-not-found']} ${injectionPoint}`);
   }
 
-  assert(injectionResults.length === 1, errors['multiple-injection-points'] +
-    options.injectionPoint);
+  assert(injectionResults.length === 1, `${errors['multiple-injection-points']}
+  ${options.injectionPoint ? options.injectionPoint : ''}`);
```

Comment:
I think this could just be `injectionPoint`, since you've previously created that variable with this value.